src/main/java/org/commonjava/storage/pathmapped/util/PathMapUtils.java

```diff
@@ -14,6 +14,16 @@
 {
     public final static String ROOT_DIR = "/";
 
+    /**
+     * This file system is not used in working entries, it is used to mark deprecated file entry which is generated
+     * in checksum depupe process. Because to do dedupe we need to generate the file by stream first and then get its
+     * checksum, so when there is dupe that means we need to clean the generated files. So to insert the pathmap with this
+     * file storage successfully(because pathmap's primary key is (filesystem, parentpath, filename), which is inserted
+     * with a successful entry in the de-dupe process firstly but used a existed file storage path, we need to create a new
+     * entry point to that deprecated file storage path created and let gc clean it later. So add this filesystem for this purpose.
+     */
+    public final static String DEDUPE_DEPRECATE_FILESYSTEM = "DEDUPE_CHECKSUM_FILESYSTEM";
```

Comment:
this seems not used?

Reply:
Yup. In old thought I'm thinking if a new pathmap of deprecated needed with this, but after checking we just need to do a reclaim but not a deprecated pathmap. Forgot to remove this.